packages/server/src/service/contact/loader.ts

```diff
@@ -17,6 +17,8 @@ export const loadState = async (state: AvailableState): Promise<[AvailableState,
 export const loadStates = async (): Promise<RawContactRecord> => {
   const startTime = new Date()
   const records = await Promise.all(availableStates.map(state => loadState(state)))
+  if (!Object.fromEntries)
```

Comment:
Hi: this was never a problem on the server.  Is there a reason we would want to add it?

Reply:
I added because to run the command `yarn bootstrap && yarn build && yarn lint && yarn test` there is an error about that function.

Doing some research seems to be about some ES version problem, that could be fixed using this code as a polyfill, I tried to change the NodeJS version using nvm, and recompile, but it didn't work out. I'll try to use an old yarn version to see if it works. But it's more like a support code than a fix (that's why I committed as a chore). But if it's only on my environment, we can remove it.